templates/terraform/pre_create/api_config.go.erb

```diff
@@ -0,0 +1,12 @@
+var apiConfigId string
```

Comment:
Instead of moving `pre_create`, can we use an `encoder` instead? That's more in line with where we've put similar functionality in the past.

Reply:
Thanks, worked nicely. I wasn't sure what it did.